torch/nn/modules/activation.py

```diff
@@ -331,7 +331,7 @@ def forward(self, input: Tensor) -> Tensor:
 
 
 class Tanh(Module):
-    r"""Applies the element-wise function:
+    r"""Applies the Hyperbolic Tangent (Tanh) function, element-wise.
```

Comment:
What about this? `Applies the Hyperbolic Tangent (Tanh) function element-wise. This function is defined as:`

Reply:
fixed